src/core/common/Topbar/index.js

```diff
@@ -8,37 +8,56 @@ import AppBar from '@material-ui/core/AppBar';
 import TopPart from "./fragments/TopPart";
 import LowPart from "./fragments/LowPart";
 
+import { demographicsAction } from "../../actions/demographocsAction";
 import { themeCommonElements } from "../../../version/config/theme.config";
 
 /**
  * This is common component for custom core TopBar
  *
  * @author Bogdan Shcherban <bsc@piogroup.net>
  */
-const CustomTopbar = props => {
-    const ThemeTopBar = get(themeCommonElements, 'topbar', false);
-    if (ThemeTopBar) {
+class CustomTopbar extends Component {
+
+    componentDidMount() {
+        const currentUserID = localStorage.getItem('userId');
+        this.props.getDemographicsAction(currentUserID);
+    }
+
+    render() {
+        const ThemeTopBar = get(themeCommonElements, 'topbar', false);
```

Comment:
again, you'll need to do an inversion of control here. Maybe in a future PR.

Reply:
It will be changed later